how-to/donate_project.md

```diff
@@ -0,0 +1,111 @@
+# Project Donation Application
+
+Please see the [proposals/new-project-donation-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Overlap with existing Kubeflow projects
+- [ ] Public list of adopters.
```

Comment:
This is not CNCF checklist, isn't ?

Reply:
No, this was just the list you sent me.

Reply:
Let's maybe place the `Overlap with existing Kubeflow projects` under Kubeflow Checklist (e.g. I would rename it from Kubeflow Specific Questions).

And keep `Public list of adopters` under CNCF checklist.

Reply:
Updated 👍